packages/backend/src/controllers/githubController.ts

```diff
@@ -47,21 +47,23 @@ export class GithubInstallController extends BaseController {
     async installGithubAppForOrganization(
         @Request() req: express.Request,
     ): Promise<void> {
-        const redirectUrl = new URL(
+        const returnToUrl = new URL(
             '/generalSettings/integrations',
             lightdashConfig.siteUrl,
         );
+        const subdomain = lightdashConfig.github.redirectDomain;
 
         const githubAppName = lightdashConfig.github.appName;
 
-        const state = req.user!.userUuid; // todo: encrypt this?
         req.session.oauth = {};
-        req.session.oauth.returnTo = redirectUrl.href;
-        req.session.oauth.state = state;
+        req.session.oauth.returnTo = returnToUrl.href;
+        req.session.oauth.state = subdomain;
+        req.session.oauth.inviteCode = req.user!.userUuid;
```

Comment:
we used to add the userUuid to the state, but we need it now for the redirect, so I moved the userUuid to the existing inviteCode in the oauth